### ISSUE TITLE
New "Wait for Confirmation" Profile

### ISSUE THREAD

**Issue by Chris6ix** [Osiris-Team/AutoPlug-Client#159]
**Importance:** 10/10

**Problem:**
I'm really interested in using AutoPlug mainly to automatically update my server jar and its plugins...the thing is the current "profiles" (NOTIFY, MANUAL and AUTOMATIC) are not quite what I need because I would want it to automatically download and install updates, but sometimes there are updates that break stuff such as for example a plugin update that requires you to generate a clean config and reconfig it...
In this situation, it would be terrible if it would update the plugin...as it would just break it...

**Solution:**
The solution to the problem would be a "Wait for Confirmation" Profile (Or something in this style) that would be a mix of the NOTIFY and AUTOMATIC Profiles.
This Profile would detect if a plugin or server jar needs an update, notify the owner about it on the web panel (With Plugin/Server Jar Name, version installed, version that it wants to update to and maybe also a link to that plugin/server jar page), so that the owner can check the update's changelog and verify that it won't break something...
The notification would then have something like a "Confirm Update" Button next to it that would when pressed activate the normal behavior of the "AUTOMATIC" Profile, which would then normally download and apply the update on the next server restart :)

If there is something that would break in a specific update, then this update can be skipped until the owner manually updates it with the required steps (Such as a reconfiguration)

Please tell me what you think of this, as I would really love to use AutoPlug :)

**Comment by Osiris-Team** [comment]
Yeah good idea. It would be more like an enhanced notify profile.
- [ ] New command to update specific plugin individually.
- [ ] Add button to execute that command on autoplug web.
